DeliBuddy/app/src/main/java/yapp/android1/delibuddy/ui/address/AddressSharedViewModel.kt

```diff
@@ -46,6 +52,9 @@ class AddressSharedViewModel @Inject constructor(
     private val _isActivate = MutableEventFlow<Boolean>()
     val isActivate: EventFlow<Boolean> = _isActivate
 
+    private var _saveAddressEvent = MutableStateFlow<SaveAddressEvent>(SaveAddressEvent.Idle)
+    val saveAddressEvent: StateFlow<SaveAddressEvent> = _saveAddressEvent
```

Comment:
이 Event들을 StateFlow말고 EventFlow로 처리해 주는게 어떨까요?
상태보다는 단발성 이벤트가 맞다는 생각이 들어서요!
LiveData에 Event클래스 래핑한거랑 똑같이 생각하시면 됩니다!

```suggestion
    private var _saveAddressEvent = MutableEventFlow<SaveAddressEvent>()
    val saveAddressEvent: EventFlow<SaveAddressEvent> = _saveAddressEvent
```